functions/ramp.h

```diff
@@ -0,0 +1,17 @@
+#ifndef FUNCTIONS_RAMP_H
+#define FUNCTIONS_RAMP_H
+
+//Returns led as value between 0 ~ 32768
+
+class Ramp {
```

Comment:
Actually, when I think about it, I think we should call it RampF<>
It makes it easy to see that it's a function, and it's much less likely to collide with something.

Reply:
updated